### PR TITLE
your-editor: 1203 -> 1206

### PR DESCRIPTION
**Pull request by uniquepointer**
Bump your-editor version

###### Motivation for this change
1206 is a stable release of `your-editor` and it will be a couple weeks before another new stable version

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/your-editor/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "your-editor";
-  version = "1203";
+  version = "1205";
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/your-editor/default.nix:5]: This should be using the unstable-XXX-XX-XX version format.

Comment (figsoda) [reply pkgs/applications/editors/your-editor/default.nix:5]: it is versioned despite not being tagged: https://github.com/kammerdienerb/yed/blob/master/src/version.h
the latest is actually 1206 now

It's probably a good idea to open an issue upstream to tag the releases so we don't have to keep track of the rev

Comment (uniquepointer) [reply pkgs/applications/editors/your-editor/default.nix:5]: Even if yed is versioned?
https://github.com/kammerdienerb/yed/blob/master/src/version.h#L4 
also it just got bumped to 1206, so is it ok to close this pr and reopen it in some hours after i test 1206?

Comment (uniquepointer) [reply pkgs/applications/editors/your-editor/default.nix:5]: Ok I will bring it up, thank you

**Comment by uniquepointer** [comment]
I rebased the commits so its just 1

File: pkgs/applications/editors/your-editor/default.nix [hunk 2]
```diff
@@ -2,13 +2,14 @@
...
+  version = "1206";
 
   src = fetchFromGitHub {
     owner = "kammerdienerb";
     repo = "yed";
-    rev = "608418f2037dc4ef5647e69fcef45302c50f138c";
-    sha256 = "KqK2lcDTn91aCFJIDg+h+QsTrl7745So5aiKCxPkeh4=";
+    rev = "6cdd99fe1359899b26d8967bd376fd5caa5451eb";
+    sha256 = "0XECSolW/xPXd1v3sv9HbJMWuHGnwCOwmHoPNCUsE+w=";
+
```
Comment (figsoda) [inline pkgs/applications/editors/your-editor/default.nix:12]:
```suggestion
```

Comment (uniquepointer) [reply pkgs/applications/editors/your-editor/default.nix]: want me to squash this as well?

Comment (figsoda) [reply pkgs/applications/editors/your-editor/default.nix]: yes, or I can squash merge, up to you

Comment (uniquepointer) [reply pkgs/applications/editors/your-editor/default.nix]: squash merge sounds better, thank you!

**Pull request merged**